workers/datasets_based/src/datasets_based/workers/config_names.py

```diff
@@ -89,9 +89,11 @@ def compute_config_names_response(
             for config in sorted(get_dataset_config_names(path=dataset, use_auth_token=use_auth_token))
         ]
     except _EmptyDatasetError as err:
-        raise EmptyDatasetError("The dataset is empty.", cause=err) from err
+        raise EmptyDatasetError("The dataset is empty. Please fix your loading script.", cause=err) from err
     except Exception as err:
-        raise ConfigNamesError("Cannot get the config names for the dataset.", cause=err) from err
+        raise ConfigNamesError(
+            "Cannot get the config names for the dataset. Please fix your loading script.", cause=err
```

Comment:
For this very specific case, I think this can only be raised when using a loading script. When using data files, the config name is just the dataset id with double hyphens (set by HubDatasetModuleFactoryWithoutScript):
```python
"config_name": self.name.replace("/", "--"),
```